src/Support/Traits/DataContainerTrait.php

```diff
@@ -118,7 +118,11 @@ public function has($key)
      */
     public function forget($key)
     {
-        Arr::forget($this->items, $key);
+        $items = $this->items;
+
+        Arr::forget($items, $key);
+
+        $this->items = $items;
```

Comment:
https://github.com/illuminate/support/blob/master/Arr.php#L193

Should `& $array` already update it by reference?